srcpkgs/alsa-ucm-conf/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'alsa-ucm-conf'
+pkgname=alsa-ucm-conf
+version=1.2.4
+revision=1
+archs="i686* x86_64*"
+depends="alsa-lib"
```

Comment:
Does it actually need to depend on `alsa-lib`? The files are necessary for correct ALSA on some places, perhaps, but maybe it shouldn't need ALSA.

Reply:
It doesn't have a runtime dependency on ALSA, no. I'll remove the line.